src/test/scala/jsonrpc4s/tests/ServicesSuite.scala

```diff
@@ -16,5 +16,6 @@ object ServicesSuite extends SimpleTestSuite {
     intercept[IllegalArgumentException] {
       base.notification(duplicate)(_ => ())
     }
+    ()
```

Comment:
I wasn't positive on the best way to do this, but `intercept` not returns the `Throwable` rather than `Unit`. I looked in minitest and they seem to also just add in a `()`, so I followed their lead here and did the same.

https://github.com/monix/minitest/blob/51a8fd22adcb0eae42e4828e31d462aa2da8a2ac/shared/src/test/scala/minitest/tests/SimpleTest.scala#L77